account_invoice_project/README.rst

```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Account Invoice Project
+=======================
+
+This module provides a new computed fields into account.invoice model
```

Comment:
Please provide a clear description. eg what field? how is it computed? why is it useful?